.circleci/config.yml

```diff
@@ -392,6 +396,7 @@ jobs:
             cd build
             ${CMAKE_BIN} -G "${CMAKE_GENERATOR}" -DCMAKE_BUILD_TYPE=Debug -DOPTDBG=1 -DPORTABLE=1 -DSNAPPY=1 -DJNI=1 << parameters.extra_cmake_opt >> ..
             cd ..
+            msbuild.exe /version
```

Comment:
Seems msbuild.exe version is always the same for all builds (either 2015 or 2019, etc.), and it already displays the version information:
```
Microsoft (R) Build Engine version 16.8.2+25e4d540b for .NET Framework
```
I don't this line is necessary, and it's causing an error line:
```
MSBUILD : error MSB1009: Project file does not exist.
```
I think we should just change it to
```
echo "Building with VS version: ${CMAKE_GENERATOR}"
```